lago/virt.py

```diff
@@ -192,31 +191,33 @@ def virt_path(self, *args):
     def bootstrap(self):
         utils.invoke_in_parallel(lambda vm: vm.bootstrap(), self._vms.values())
 
-    def export_entities(self, entities_names, standalone, dst_dir, compress):
-        if not entities_names:
-            entities_names = self._vms.keys()
+    def export_vms(self, vms_names, standalone, dst_dir, compress):
+        if not vms_names:
+            vms_names = self._vms.keys()
 
-        running_ents = []
-        entities = []
-        for name in entities_names:
+        running_vms = []
+        vms = []
+        for name in vms_names:
             try:
-                ent = self._vms[name]
-                entities.append(ent)
-                if ent.defined():
-                    running_ents.append(ent)
+                vm = self._vms[name]
+                vms.append(vm)
+                if vm.defined():
+                    running_vms.append(vm)
             except KeyError:
-                LOGGER.error('Entity {} does not exist'.format(name))
-                sys.exit(1)
+                raise utils.LagoUserException(
+                    'Entity {} does not exist'.format(name)
+                )
 
-        if running_ents:
-            for ent in running_ents:
-                LOGGER.error('{} must be off'.format(ent.name()))
-            sys.exit(1)
+        if running_vms:
+            raise utils.LagoUserException(
+                'The following vms must be off:\n{}'.
+                format('\n'.join([_vm.name() for _vm in running_vms]))
+            )
         # TODO: run the export task in parallel
 
         with LogTask('Exporting disks to: {}'.format(dst_dir)):
-            for ent in entities:
-                ent.export_disks(standalone, dst_dir, compress)
+            for _vm in vms:
```

Comment:
Otherwise the formatter is complaining about the re-assignment of varaible "vm" . I just needed another name for the variable.